public/src/js/widgets/config-collection-tags.js

```diff
@@ -0,0 +1,55 @@
+import BaseWidget from 'widgets/base-widget';
+import $ from 'jquery';
+import ko from 'knockout';
+import _ from 'underscore';
+import 'fastsearch';
+import 'fastselect';
+
+export default class CollectionTags extends BaseWidget {
+    constructor(params) {
+        super();
+
+        this.tags = params.tags;
+        const tags = params.tags();
+
+        var initialInput = _.pluck(tags, 'type').join(',');
+
+        this.inputTags = ko.observable(initialInput);
+
+        var initialValues = _.map(tags, tag => {
+            return{
+              'text': tag.type,
+              'value': tag.type
+
+            };
+        });
+
+        $('.multipleInputDynamic').attr('data-initial-value', JSON.stringify(initialValues));
```

Comment:
this applies to all `.multipleInputDynamic` in the page, you need to target it to a specific element. Look into [custom bindings](http://knockoutjs.com/documentation/custom-bindings.html)